docs/src/pages/guide/how-to/02-adding-tools.mdx

```diff
@@ -77,6 +77,83 @@ export const mastra = new Mastra({
 });
 ```
 
+## Debugging Tools
```

Comment:
This is too long. Two sentences and a link to the example is fine. "You can test tools using Jest. Here's an example [link]"